source/guide_maildrop-autoresponder.rst

```diff
@@ -0,0 +1,260 @@
+.. highlight:: console
+
+.. author:: Achim | pxlfrk <hallo@pxlfrk.de>
+
+.. tag:: mail
+.. tag:: bouncer
+.. tag:: maildrop
+.. tag:: qmail
+.. tag:: courier
+
+
+
+.. sidebar:: About
+
+  .. image:: _static/images/maildrop_autoreply.svg
+      :align: center
+
+######################
+Maildrop Autoresponder
+######################
+
+.. tag_list::
+
+The following guide allows you to enable a custom autoresponder for a specific mail address by using the built-in maildrop_ in addition to mailbot_, qmail_ and some tweaks.
+
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`domains <mail-domains>`
+  * :manual:`mailboxes <mail-mailboxes>`
+  * :manual:`webmail <mail-access>`
+
+
+Prerequisites
+=============
+
+If you're planning to use your own domain for mail with your Uberspace, you need to set it up before. Refer to the :manual:`manual <mail-domains>` and come back when your done.
+
+Installation
+============
+
+For this guide, let's assume that you have created a mail user ``email``. We'll also assume that your EMail address is therefore ``email@stardust.uber.space``. If you're using an external domain, e.g. ``domain.tld``, refer to it accordingly. Make sure to replace these placeholder tags with your own credentials and usernames.
+
+Create the autoreply-filter
+---------------------------
+
+Create a directory to store your configurations and keep your home directory nice and clean. Use your favourite editor to create ``~/autoresponder/.autoreply-filter`` with the following content. We'll split the content in multiple parts to explain what they're doing. Make sure you don't miss one, otherwise the filter won't work!
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ mkdir autoresponder
+ [isabell@stardust autoresponder]$  
+
+.. code-block:: ini
+
+	# set default Maildir
+	MAILDIR="$HOME/Maildir"
+	
+	# check if we're called from a .qmail-EXT instead of .qmail
+	# If you're using a separate mail user (not the primary), we setup the MAILDIR accordingly
+	import EXT
+	if ( $EXT )
+	{
+		# does a vmailmgr user named $EXT exist?
+		# if yes, deliver mail to their Maildir instead
+		CHECKMAILDIR = `dumpvuser $EXT | grep '^Directory' | awk '{ print $2 }'`
+		if ( $CHECKMAILDIR )
+		{
+			MAILDIR="$HOME/$CHECKMAILDIR"
+		}
+	}
+	
+The variable ``$MAILDIR`` defines where maildrop finds the maildir and what it is. The example checks whether the call is for a specific mailbox or the main mailbox.
+
+.. code-block:: ini
+
+	# If you want to debug the autoresponder, turn on logging by uncommenting the following line.
+	logfile "$HOME/autoresponder/autoreply-filter.log"
```

Comment:
The line is not commented out so it cannot be uncommented, please rephrase.

Reply:
rephrased, thanks for the hint!